ios/src/TGMapViewController.h

```diff
@@ -45,12 +46,17 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)mapView:(TGMapViewController *)view recognizer:(UIGestureRecognizer *)recognizer didRecognizeShoveGesture:(CGPoint)location;
 @end
 
+NS_ASSUME_NONNULL_END
+
 @protocol TGMapViewDelegate <NSObject>
 @optional
-- (void)mapView:(TGMapViewController*)mapView didLoadSceneAsync:(NSString*)scene;
-- (void)mapView:(TGMapViewController*)mapView didSelectFeature:(NSDictionary*)feature atScreenPosition:(CGPoint)position;
+- (void)mapView:(TGMapViewController *)mapView didLoadSceneAsync:(nonnull NSString *)scene;
+- (void)mapView:(TGMapViewController *)mapView didSelectFeature:(NSDictionary *)feature atScreenPosition:(CGPoint)position;
```

Comment:
Since there's a need for a `NS_ASSUME_NONNULL`, my guess was that the default behavior for those were `nullable`, I don't want bloat signatures of function parameters with `nullable` everywhere except if it happens to actually be `nonnullable` by default.